readthedocs/api/v2/views/footer_views.py

```diff
@@ -102,18 +102,23 @@ def _get_version(self):
             if version_slug == '':
                 version_slug = LATEST
 
+            project = self._get_project()
             version = get_object_or_404(
-                Version.objects.public(
-                    user=self.request.user,
-                    project=self._get_project(),
-                    only_active=False,
-                ),
+                project.versions.all(),
                 slug__iexact=version_slug,
             )
             setattr(self, cache_key, version)
 
         return version
 
+    def _get_all_versions_sorted(self):
```

Comment:
This name is confusing.

It says that returns all versions, but then it calls "ordered_active_versions" which I suppose that returns only active versions.

Reply:
What do you suggest?

Reply:
`_get_active_versions_sorted`